python/startup/bootstrap.py

```diff
@@ -1,47 +1,66 @@
 # Copyright (c) 2016 Shotgun Software Inc.
-# 
+#
 # CONFIDENTIAL AND PROPRIETARY
-# 
-# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit 
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
 # Source Code License included in this distribution package. See LICENSE.
-# By accessing, using, copying or modifying this work you indicate your 
-# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights 
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
```

Comment:
we should add a comment here explaining what this entry point is and who is calling it.

Am i right thinking that this is now just a legacy pathway for the old tk-multi-launchapp? 
We should add some notes to say that this will be deprecated at some point in the future when noone is using the older launchapp.